cloudbuild.yaml

```diff
@@ -0,0 +1,7 @@
+# Set gcloubuild trigger for master branch only
+steps:
+- id: appengine_deployment 
+  name: 'gcr.io/cloud-builders/gcloud' 
+  args: ['app', 'deploy'] 
+  timeout: '600s'
+  
```

Comment:
nit: editor still not right. you can be a perfectionist or you can just merge now.

Reply:
I can't believe that, apologies. Perfectionism limit reached, will fix up next time sorry